bigbluebutton-html5/imports/ui/stylesheets/variables/palette.scss

```diff
@@ -24,3 +24,5 @@ $color-gray-label: $color-gray !default;
 $color-blue-light: lighten($color-primary, 35%);
 $color-blue-lighter: #92BCEA;
 $color-blue-lightest: #E4ECF2;
+
+$color-fade: rgba(243, 246, 249, 0);
```

Comment:
Why not convert to hexadecimal?

Reply:
We can't store the alpha in hexadecimal, and eight-digit color codes are not supported on all browsers.

Reply:
But you are setting the alpha to 0 thus is the same as `transparent`